src/app/(authenticated)/search/page.tsx

```
@@ -326,7 +370,25 @@ export default function SearchProfilesPage() {
     );
   }
 
+  // Visual on-page banner to verify the page truly mounted even if console logs are filtered
+  const [mounted, setMounted] = React.useState(false);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)